.github/workflows/build-images.yaml

```diff
@@ -59,12 +71,23 @@ jobs:
     needs: rust-images
     uses: ./.github/workflows/sdk-integration-test.yaml
     secrets: inherit
+    with:
+      GIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
is it possible to do:

```
with:
  GIT_SHA: ${{ .env.GIT_SHA }}
```

so we can reuse the env defined at the top of the workflow?

Reply:
just tried it out. unfortunately we can't - https://github.com/aptos-labs/aptos-core/actions/runs/2450658591 . Somehow `env` is not available in the context of calling another workflow.